src/components/TextField/index.tsx

```diff
@@ -20,10 +21,16 @@ export interface TextFieldProps
   iconLeft?: React.ReactNode;
   iconRight?: React.ReactNode;
   styleType?: Style;
+  className?: string;
 }
 
-const generateClassName = ({ errors, iconLeft, styleType }: IClassNameArgs) => {
-  const baseClass = "input__field";
+const generateClassName = ({
+  errors,
+  iconLeft,
+  styleType,
+  className,
```

Comment:
We should additionally guard the `className` arg here to avoid passing an `undefined` value to the class name.
```suggestion
  className = "",
```